planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -673,6 +676,18 @@ BehaviorModuleOutput PullOverModule::plan()
     goal_pose_pub_->publish(goal_pose_stamped);
   }
 
+  uint16_t direction = SteeringFactor::STRAIGHT;
+  if (getTurnInfo().first.command == TurnIndicatorsCommand::ENABLE_LEFT) {
+    direction = SteeringFactor::LEFT;
+  } else if (getTurnInfo().first.command == TurnIndicatorsCommand::ENABLE_RIGHT) {
+    direction = SteeringFactor::RIGHT;
+  }
+  // TODO(tkhmy) add handle status TRYING
+  steering_factor_interface_ptr_->updateSteeringFactor(
+    {getParkingStartPose(), modified_goal_pose_},
```

Comment:
@kosuke55 If I want to get the goal pose, which one will be more accurate?
modified_goal_pose_ or calcRefinedGoal() ?

Reply:
`modified_goal_pose_` is more accurate.
This is the final processed goal.

Reply:
The goal processing sequence is 
`planner_data_->route_handler->getGoalPose()` -> `calcRefinedGoal()` -> `modified_goal_pose_`